javaparser-core/src/main/java/com/github/javaparser/UnicodeEscapeProcessingProvider.java

```diff
@@ -110,7 +114,7 @@ private void expandBuffer(boolean wrapAround) {
                 bufpos -= tokenBegin;
             }
         } catch (Exception t) {
-            throw new RuntimeException(t.getMessage());
+            throw new RuntimeException(t.getMessage(), t);
```

Comment:
This whole class has been copied from JavaCC. There is an issue to completely refactor it because it is a horror.